src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator</RootNamespace>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
@sharwell had the same question too -- I had kept it on desktop because I was under the impression that loading windows projects on CoreCLR was still fairly busted with MSBuild. That _might_ be better now, but the icky bit is MSBuildWorkspace has to load types for MSBuild tasks, and those are still desktop only in some cases. Now that we've had other projects go down enabling both for nullable support, I guess I can just do that -- there's just not necessarily promises you want to use the CoreCLR one...